cmake/FindProtobuf.cmake

```diff
@@ -1,126 +1,685 @@
-#
-# Locate and configure the Google Protocol Buffers library
-#
-# Adds the following targets:
-#
-#  protobuf::libprotobuf - Protobuf library
-#  protobuf::libprotobuf-lite - Protobuf lite library
-#  protobuf::libprotoc - Protobuf Protoc Library
-#  protobuf::protoc - protoc executable
-#
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
```

Comment:
This file is installed together with a protobuf library under `/usr/share/cmake-3.20/Modules/FindProtobuf.cmake`
No reason to have it in our source tree